README.md

```diff
@@ -223,36 +416,125 @@ in subdirectories per architecture:
+* source packages are on top of this directory
+* `noarch`, `x86_64` and `aarch64` directories contains final packages
+
+## Preparing Red Hat system before installation
+Here are notes for installation compiled packages on target system.
+
+### RHEL Repositories
+CodeReady and EPEL have to be  enabled before installation EPEL is required only
+for mbed-edge-example package.
+
+```bash
+sudo subscription-manager repos --enable codeready-builder-for-rhel-8-x86_64-rpms
+sudo yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
+```
+
+### Docker
+RHEL 8 does not support Docker, to  use kubelet it is required to install Docker
+from  external repository.  This is  required only  to run  built binaries  (not
+required to do the actual build). To add the repository, run:
+
+```bash
+sudo yum-config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo
+```
+
+### SELinux
+SELinux is not  currently supported: `pelion-relay-term` service  will be killed
+when SELinux is enabled.
+
+SELinux  has to  be  disabled, set  to  permissive or  `node`  binary should  be
+excluded. More details about SELinux and its configuration here:
+https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/using_selinux/index
+
 ## Installing packages
 
-Build results can be installed onto a target system either by manually
-copying the packages to a target system and installing via apt or by
-making the packages available in an APT repository on a server and installing
-on the target via apt. The instructions in this section show how to install
-the packages manually.  See the next section for setting up an APT repository.
+Build results can  be installed onto a target system  either by manually copying
+the packages  to a  target system and  installing via apt/yum  or by  making the
+packages available  in an package repository  on a server and  installing on the
+target via  apt/yum. The instructions  in this section  show how to  install the
+packages  manually. See  the  next  section for  setting  up  an APT  repository
+(Debian/Ubuntu only).
+
+### Installing on Debian or Ubuntu
 
-Copy the debian packages found in `build/deploy/deb/` to the
-target system and install with `apt`.
+Copy  the debian  packages found  in `build/deploy/deb/<DISTRO>`  to the  target
+system and install with `apt`.
 
 Install all packages with the following command.
-```
+```bash
 $ sudo apt install -y ./*.deb
 ```
 
 Or, install a single package by specifying its deb file name.
-```
+```bash
 $ sudo apt install -y ./devicedb_<version>_<arch>.deb
 ```
 
-## Removing packages
+### Installing on Red Hat or Centos
+1.  Before installing  packages make  sure  that you  have subscription  enabled
+and   EPEL,  CodeReady   and  Docker   repositories  are   enabled  (see   [RHEL
+repositories](#rhel-Repositories) and [Docker](#docker)).
+
+2.      Copy     content      of     `build/deploy/rpm/<distro>/<arch>`      and
```

Comment:
formatting for this one is off

Reply:
Fixed. I'm using `par jw80` command to justify text, but github detects mutiple spaces after '2.' as code block (but for example Typora does not).